### PR TITLE
Update 279_sexshop

### PR DESCRIPTION
**Pull request by k9loverjasmine**
Change text cleaning up the grammar and word choice.

### CODE REVIEW

File: locations/279_sexshop [hunk 1]
```diff
@@ -1,4 +1,3 @@
-# sexshop
```
Comment (morgolon) [inline locations/279_sexshop]: i think this line is essential and removing it will break the game

File: locations/279_sexshop [hunk 2]
```diff
@@ -732,18 +731,16 @@ cla
...
-	'Approaching the mirror you are considering this so-called skirt seller kneels behind you and starts praising naglazhivat your ass skirt, his hands already wander around your hips and lips as close to your buttocks says that when they touch on your skin, word soon cease and remain just the touch of his lips to your skin, closer to prilizhayuschiesya pussy.'
-	act 'Push him': gt 'sexshop','start'
+	'You wear the skirt out of the booth and pose in the mirror, you do not have panties on and the skirt is so short that looks more like a belt and does not cover anything.  All your charms are visible from all directions.'
+	'Approaching the mirror you are considering this so-called skirt, and the cashier kneels behind you and starts praising your ass in that skirt, his hands already wander around your hips and he get his lips as close to your butt as he can, he says that he wants to kiss on your skin.  Words soon cease and just the touch of his lips to your skin remains, closer and closer to your pussy.'
+	act 'Push him away': gt 'sexshop','start'
 	act 'Let him': gt 'sex','kuni'
 	end
 end
---- sexshop ---------------------------------
```
Comment (morgolon) [inline locations/279_sexshop]: same with this line

**Comment by morgolon** [comment]
Looks really good! Do you mind adding the two lines I pointed out back in? They're the brackets for a location and I'm pretty sure they're needed for the sex shop to work at all.

File: locations/279_sexshop (hunk 1 above)
Comment (k9loverjasmine) [inline locations/279_sexshop]: Will edit when can get on comp next ussual on phone and wife has comp.
Will try tom.

Comment (morgolon) [inline locations/279_sexshop]: No need, I already merged it in and fixed this line :)

**Pull request merged**